.github/workflows/reusable_on_release.yaml

```diff
@@ -0,0 +1,122 @@
+name: Release pipeline
+
+# Example usage from the caller workflow:
+#
+# It requires the following secrets (inheriting them is recommended for simplicity)
+#  - OHAI_AWS_ROLE_ARN_PRODUCTION
+#  - OHAI_AWS_ACCESS_KEY_ID_PRODUCTION
+#  - OHAI_AWS_SECRET_ACCESS_KEY_PRODUCTION
+#  - OHAI_GPG_PASSPHRASE
+#  - OHAI_GPG_PRIVATE_KEY_BASE64
+#  - COREINT_SLACK_TOKEN
+#  - COREINT_SLACK_CHANNEL
+
+# jobs:
+#   job_name:
+#     uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
+#     with:
+#       tag: ${{ github.event.release.tag_name }}
+#       integration_name: "integration-name" # without the 'nri-' prefix
+#       # test_package: false # this will skip the package testing, useful when the integration is included with the agent (nri-docker)
+#       # setup a schema for the publish action different from "ohi"
+#       # publish_schema: "custom"
+#       # setup a custom schema url (requires publish_schema='custom')
+#       # publish_schema_url: "https://raw.githubusercontent.com/newrelic/nri-${{ env.INTEGRATION }}/${{ env.TAG }}/build/s3-publish-schema.yml"
+#     secrets: inherit
+
+on:
+  workflow_call:
+
+    inputs:
+      tag:
+        description: Tag to be released
+        type: string
+        required: true
+      integration_name:
```

Comment:
nit: #41 uses `integration` for this variable, I would change it to be consistent